packages/theme/src/theme.ts

```diff
@@ -62,6 +62,11 @@ export enum SupportedLanguages {
     zhTW = 'zh-TW',
```

Comment:
Is it possible to consider putting SupportedLanguages  into shared

Reply:
I'll move it into `public-apis` package.